packages/web-components/fast-components/src/design-tokens.ts

```diff
@@ -256,24 +259,41 @@ export const neutralFillStrongFocusDelta = create<number>(
 /** @public */
 export const neutralFillLayerRestDelta = create<number>(
     "neutral-fill-layer-rest-delta"
-).withDefault(3);
+).withDefault(2);
 
 /** @public */
 export const neutralStrokeRestDelta = create<number>(
     "neutral-stroke-rest-delta"
-).withDefault(25);
+).withDefault(49);
 /** @public */
 export const neutralStrokeHoverDelta = create<number>(
     "neutral-stroke-hover-delta"
-).withDefault(40);
+).withDefault(49);
 /** @public */
 export const neutralStrokeActiveDelta = create<number>(
     "neutral-stroke-active-delta"
-).withDefault(16);
+).withDefault(90);
 /** @public */
 export const neutralStrokeFocusDelta = create<number>(
     "neutral-stroke-focus-delta"
-).withDefault(25);
+).withDefault(49);
+
+/** @public */
+export const neutralStrokeInputFilledRestDelta = create<number>(
```

Comment:
I don't see these used here. Are they related to another component perhaps? This is fine, but I want to be cautious about naming here. We want these recipes to be as common or generic as reasonably possible. For example, is this intended to be paired with the "secondary" fill? If so, could we call this "secondary" stroke instead? Unfortunately the names don't always line up and aren't always used in pairs, but a very specific name becomes hard to use or abused easily.

Reply:
removed